docs/zh/engines/database-engines/replicated.md

```diff
@@ -0,0 +1,116 @@
+# [experimental] Replicated {#replicated}
+
+该引擎基于[Atomic](../../engines/database-engines/atomic.md)引擎。它支持通过将DDL日志写入ZooKeeper并在给定数据库的所有副本上执行的元数据复制。
+
+一个ClickHouse服务器可以同时运行和更新多个复制的数据库。但是同一个复制的数据库不能有多个副本。
+
+## 创建数据库 {#creating-a-database}
+
+``` sql
+CREATE DATABASE testdb ENGINE = Replicated('zoo_path', 'shard_name', 'replica_name') [SETTINGS ...]
+```
+
+**引擎参数**
+
+-   `zoo_path` — ZooKeeper地址，同一个ZooKeeper路径对应同一个数据库。
+-   `shard_name` — 分片的名字。数据库副本按`shard_name`分组到分片中。
+-   `replica_name` — 副本的名字。同一分片的所有副本的副本名称必须不同。
+
+!!! note "Warning"
```

Comment:
Quoted words should be translated as the reader see them -> `!!! note "(translation of warning)"`.
Also, it is strange that in one place you have used warning and in another note with "Warning".